old-fashioned-image-build

```diff
@@ -130,7 +130,14 @@ lxc exec lp-$SERIES-amd64 -- hostname standalone-lp-$SERIES-amd64
 
 # Inject the files from the current tree in the right place in the LXD
 # container.
-lxc exec lp-$SERIES-amd64 -- mkdir /usr/share/livecd-rootfs
+# First install livecd-rootfs. We will replace the bulk of the livecd-rootfs
+# code with our local copy
```

Comment:
"without this livecd-rootfs is installed by buildlivefs (below) overwriting changes"